### PR TITLE
Update README and .travis.yml to support Codecov for code coverage

### PR DESCRIPTION
**Pull request by Mackarous**
#5 Update README and .travis.yml

### CODE REVIEW

**Comment by Mackarous** [comment]
@a-mehta @DeanoDee

**Comment by Mackarous** [comment]
FYI builds will fail until the Podspec is updated with proper info

File: README.md [hunk 1]
```diff
@@ -1,33 +1,65 @@
...
+
 ## Installation
 
 ExpandableImageView is available through [CocoaPods](http://cocoapods.org). To install
 it, simply add the following line to your Podfile:
 
-```ruby
+```
 pod "ExpandableImageView"
 ```
```
Comment (DeanoDee) [inline README.md]: I think we should include the other parts of the api here api here as per our milestones:
setTransition(TransitionManager) => custom object
setSharable(boolean)
setExpandedViewController(UIViewController)

you could say coming soon for transition

**Comment by Mackarous** [comment]
@DeanoDee @a-mehta updated

**Comment by a-mehta** [comment]
Seems legit.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,52 @@
...
 
+##### API Capabilities:
+
+- [x] Expandable image views:
+```swift
+imageView.expandable = true // (default = false)
+```
+- [x] Shareable when expanded:
+```swift
+imageView.shareableWhenExpanded = false // (default = true)
```
Comment (DeanoDee) [inline README.md]: you might note that by default it's sharable above the code block

File: README.md [hunk 3]
```diff
@@ -1,16 +1,52 @@
...
+```
+- [ ] Custom transitioning:
```
Comment (DeanoDee) [inline README.md]: this is not available yet so should it be in here

File: README.md [hunk 4]
```diff
@@ -1,16 +1,52 @@
...
+imageView.transition = .Custom(MyTransition()) // (default = nil)
+```
+- [ ] Custom expanded view controller:
+```swift
+imageView.expandedViewController = MyViewController() // (default = nil)
+```
+
 ## Requirements
 
+- iOS 8+
```
Comment (DeanoDee) [inline README.md]: we are targeting 9 is this true

Comment (Mackarous) [inline README.md]: Oops! Will fix

**Pull request merged**